server/channels/app/plugin_install.go

```
@@ -323,6 +328,12 @@ func (ch *Channels) installExtractedPlugin(manifest *model.Manifest, fromPluginD
 		return nil, model.NewAppError("installExtractedPlugin", "app.plugin.install.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
 	}
 
+	// Check plugin id is not blocked
+	if plugin.PluginIDIsBlocked(manifest.Id) {
+		mlog.Debug("Skipping installation of plugin since plugin is on blocklist", mlog.String("plugin_id", manifest.Id))
+		return nil, nil
```

Should we return an explicit error here in case it's a user trying to do this?